experiments/golden-results/StratoX-summary.txt

```diff
@@ -149,9 +144,9 @@ Error message: Unknown tree node
 Nkind: N_Compilation_Unit
 --
 Occurs: 1 times
-Calling function: Do_Modular_Type_Definition
-Error message: can not handle modular types with non-power-of-2 bounds at the moment
-Nkind: N_Modular_Type_Definition
+Calling function: Do_Constant
+Error message: Constant Type not in Class_Bitvector_Type
+Nkind: N_Integer_Literal
```

Comment:
I honestly have no idea what this is. Seems like we get a literal 0 **as a type** here for some reason? I don't _think_ this is caused by this PR, but I can't be certain.

Reply:
The report comes from `a-tags.adb` where they declare a range type (which we do not support yet). Then they create constant of that type (which the symbol table does not know). And thus the type is not in `Class_Bitvector_Type`. A bit more magic will have to take place to fix it here, but this is not caused by this PR.

Reply:
Thanks! I'd seen that file but I couldn't figure out it was related to range types (wait, what do you mean we don't support those? Isn't that just normal integer types?)